ChatApp/ChatViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  ViewController.swift
+//  chatApp.test
+//
+//  Created by Ana Dzebniauri on 14.04.23.
+//
+
+import UIKit
+
+class ChatViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    //MARK: - Properties
+    private let receiverTableView = UITableView().forAutoLayout() //tag = 1
+    private let senderTableView = UITableView().forAutoLayout() //tag = 2
+
+    private let chatView = ChatView().forAutoLayout()
+    
+    //MARK: - ViewDidLoad
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        //Do any additional setup after loading the view.
+        
+        setUpChatView()
+        setUpReceiverTableView()
+        setUpSenderTableView()
+
+        receiverTableView.register(RecipientTableViewCell.self, forCellReuseIdentifier: "receiverTableViewCell")
+        senderTableView.register(SenderTableViewCell.self, forCellReuseIdentifier: "senderTableViewCell")
+    }
+    
+    //MARK: - Chat View Method
+    private func setUpChatView() {
+        view.addSubview(chatView)
+        chatView.stretchOnParent()
+    }
+
+//    MARK: - Receiver Table View Set Up
+    private func setUpReceiverTableView() {
+        receiverTableView.delegate = self
+        receiverTableView.dataSource = self
+        
+        view.addSubview(receiverTableView)
+        NSLayoutConstraint.activate([
+            receiverTableView.topAnchor.constraint(equalTo: chatView.switcherView.bottomAnchor),
+            receiverTableView.bottomAnchor.constraint(equalTo: chatView.receiverTypingArea.topAnchor),
+            receiverTableView.leftAnchor.constraint(equalTo: view.leftAnchor),
+            receiverTableView.rightAnchor.constraint(equalTo: view.rightAnchor)
+        ])
+    }
+    
+    //MARK: - Sender Table View Set Up
+    private func setUpSenderTableView() {
+        senderTableView.delegate = self
+        senderTableView.dataSource = self
+        
+        view.addSubview(senderTableView)
+        
+        NSLayoutConstraint.activate([
+            senderTableView.topAnchor.constraint(equalTo: chatView.dividerView.bottomAnchor),
+            senderTableView.bottomAnchor.constraint(equalTo: chatView.senderTypingArea.topAnchor),
+            senderTableView.leftAnchor.constraint(equalTo: view.leftAnchor),
+            senderTableView.rightAnchor.constraint(equalTo: view.rightAnchor)
+        ])
+    }
+    
+    //MARK: - Table Views
+    
```

Comment:
here I would make extension for `ChatViewController` and add delegate and datasource methods in it. 

```suggestion
    extension ChatViewController: UITableViewDelegate, UITableViewDataSource {}
```